galsim/phase_screens.py

```diff
@@ -0,0 +1,623 @@
+import numpy as np
```

Comment:
Need license and file-level docstring.  (Unless this was only for the PR and you plan to re-assemble them?)